definitions/firefox_desktop.toml

```diff
@@ -298,6 +344,24 @@ description = """
 owner = ["bochocki@mozilla.com", "firefox-kpi@mozilla.com"]
 deprecated = true
 
+[metrics.daily_active_users_glean]
+data_source = "baseline_clients_daily"
+select_expression = "COUNTIF(browser_engagement_active_ticks > 0 AND COALESCE(browser_engagement_uri_count,0) > 0)"
```

Comment:
Since the new active users tables are ready, we should probably just use those, no reason to duplicate the old version from legacy

Reply:
This is outdated metrics and not one of the guradrail metrics. I am reverting the change.